composer.json

```diff
@@ -12,9 +12,11 @@
     ],
     "require": {
         "php": ">=7.1",
-        "keboola/db-extractor-common": "~9.1",
-        "keboola/php-temp": "^1.0",
-        "keboola/php-csv-db-import": "~2.3.0"
+        "keboola/db-extractor-common": "dev-ondra-severance-repository",
+        "keboola/db-extractor-logger": "dev-ondra-init-repo",
+        "keboola/db-extractor-ssh-tunnel": "dev-ondra-init-repo",
```

Comment:
Takže tohle vyhodit, až bude releasnuto. (píšu to jen aby to bylo vidět v posledním review)

Reply:
vyhozeno